Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(url: "https://github.com/aws-geospatial/amazon-location-mobile-auth-sdk-ios.git", from: "0.2.0")
+        .package(url: "https://github.com/aws-geospatial/amazon-location-mobile-auth-sdk-ios", branch: "0.2.2")
```

Comment:
We will need to merge https://github.com/aws-geospatial/amazon-location-mobile-auth-sdk-ios/pull/10 and create a release before merging this current PR